keyserver/README.md

```diff
@@ -0,0 +1,30 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+Generate new config file with proper format
+```
+go run utils/configen/mkconfig.go
+```
+
+Generate new key pair for signing
+``` 
+go run utils/keygen/mkkey.go -vrf -signing
```

Comment:
Thanks! I fixed it in f82d43c83ac69eabb5024122556c779addca2933. It should generate both keys now.